bazel/test_runners/qemu_with_kernel/launcher.sh

```diff
@@ -0,0 +1,128 @@
+#!/bin/bash
+# Copyright 2018- The Pixie Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# SPDX-License-Identifier: Apache-2.0
+
+set -e
+
+KERNEL_IMAGE=%kernelimage%
+DISK_IMAGE=%diskimage%
+
+# Location of our utlity script we need to launch the test runner.
+bazel_qemu_test_runner_path="${PWD}/bazel/test_runners/qemu_with_kernel/test_runner_inside_qemu.sh"
```

Comment:
Shouldn't you use https://github.com/bazelbuild/bazel/blob/master/tools/bash/runfiles/runfiles.bash?